### PR TITLE
[devtools-translate] Chinese - What's New in DevTools (Chrome 109)

### PR DESCRIPTION
**Pull request by liuliangsir**
Fixes: https://github.com/GoogleChrome/developer.chrome.com/issues/4915

This PR adds Simplified Chinese translations for "What's New In DevTools (Chrome 109)". https://developer.chrome.com/blog/new-in-devtools-109/

### CODE REVIEW

**Comment by liuliangsir** [comment]
@xyugroup, @aquaMAX, @louisyoong, @hanselfmu cc

File: site/zh/blog/new-in-devtools-109/index.md [hunk 1]
```diff
@@ -27,88 +26,118 @@ draft: true
...
+在 Windows 和 Linux 系统里面，使用 <kbd>Control</kbd> + <kbd>Page up</kbd> 或 <kbd>Page down</kbd>
 
 <!-- Moreover, you can navigate the autocomplete suggestions with <kbd>Ctrl</kbd> + <kbd>N</kbd> and <kbd>Ctrl + P</kbd> on MacOS, similar to [Emacs](https://www.gnu.org/software/emacs/). For example, you can type `window.` in the `Console` and use these shortcuts to navigate. -->
+此外，您可以在 MacOS 系统里面使用 <kbd>Ctrl</kbd> + <kbd>N</kbd> 和 <kbd>Ctrl + P</kbd> 来上下选择自动填充建议，类似于 [Emacs](https://www.gnu.org/software/emacs/)。例如，您可以在 `Console` 面板里面输入 `window.` 并使用这些快捷键来上下选择自动填充建议。
 
 <!-- On top of that, DevTools now accepts <kbd>Arrow Right</kbd> for autocompletion only at the end of line. For example, an autocomplete dialog shows when you are editing something in the middle of the code. When you press the <kbd>Arrow Right</kbd> key, most likely, you want to set the cursor to the next position instead of autocomplete. This UX change better aligns with your authoring workflow. -->
+此外，DevTools 现在只会在行尾响应用于自动填充的 <kbd>Arrow Right</kbd>。例如，当您在代码的中间位置编辑某些内容时，会弹出自动填充对话框。当您按下 <kbd>Arrow Right</kbd>时，您很可能想要将光标设置到下一个位置，而不是自动填充。这个 UX 上的变动可以更好地服务于您的工作流。
```
Comment (aquaMAX) [inline site/zh/blog/new-in-devtools-109/index.md:88]:
```suggestion
此外，DevTools 现在只会在行尾响应用于自动填充的 <kbd>Arrow Right</kbd>。例如，当您在代码的中间位置编辑某些内容时，会弹出自动填充对话框。当您按下 <kbd>Arrow Right</kbd>时，您很可能想要将光标设置到下一个位置，而不是自动填充。这个 UX 上的变动可以更好地服务于您的工作流程。
```

**Review by aquaMAX — COMMENTED** [review]
just 1 minor wording

File: site/zh/blog/new-in-devtools-109/index.md (hunk 1 above)
Comment (liuliangsir) [reply site/zh/blog/new-in-devtools-109/index.md:88]: Okay

File: site/zh/_partials/devtools/whats-new.md [hunk 2]
```diff
@@ -4,16 +4,15 @@
...
-* [Miscellaneous highlights](/zh/blog/new-in-devtools-109/#misc)
-* [[Experimental] Enhanced UX in managing breakpoints](/zh/blog/new-in-devtools-109/#breakpoint-redesign)
-* [[Experimental] Automatic in-place pretty print](/zh/blog/new-in-devtools-109/#pretty-print)
- #}
+* [Recorder: 为步骤提供新的复制选项，支持页面内重放，步骤支持鼠标右键弹出菜单](/zh/blog/new-in-devtools-109/#recorder)
+* [Performance 面板里面的记录显示实际的函数名称](/zh/blog/new-in-devtools-109/#performance)
+* [Console & Sources 面板新增快捷键](/zh/blog/new-in-devtools-109/#keyboard-shortcuts)
+* [改善 JavaScript 调试体验](/zh/blog/new-in-devtools-109/#debugging)
+* [其他的更新](/zh/blog/new-in-devtools-109/#misc)
+* [【实验】优化断点管理界面的 UX](/zh/blog/new-in-devtools-109/#breakpoint-redesign)
```
Comment (jecfish) [inline site/zh/_partials/devtools/whats-new.md:14]:
```suggestion
* [【实验阶段】优化断点管理界面的 UX](/zh/blog/new-in-devtools-109/#breakpoint-redesign)
```

File: site/zh/_partials/devtools/whats-new.md [hunk 3]
```diff
@@ -4,16 +4,15 @@
...
+* [【实验】自动格式化](/zh/blog/new-in-devtools-109/#pretty-print)
```
Comment (jecfish) [inline site/zh/_partials/devtools/whats-new.md:15]:
```suggestion
* [【实验阶段】自动格式化](/zh/blog/new-in-devtools-109/#pretty-print)
```

File: site/zh/blog/new-in-devtools-109/index.md [hunk 4]
```diff
@@ -4,17 +4,16 @@ title: "DevTools 新功能（Chrome 109）"
 authors:
   - jecelynyeen
 date: 2023-01-15
-description: 'Copy step as script in the Recorder, actual function names in performance’s recordings, and more.'
+description: 'Recorder 面板支持复制步骤为脚本，performance 面板里面的记录显示真实的函数名称，以及更多。'
```
Comment (jecfish) [inline site/zh/blog/new-in-devtools-109/index.md:7]:
```suggestion
description: 'Recorder 面板支持复制步骤为脚本，在 Performance 面板里面显示真实的函数名称，以及更多。'
```

File: site/zh/blog/new-in-devtools-109/index.md [hunk 5]
```diff
@@ -27,88 +26,118 @@ draft: true
 
 
 <!-- ## Recorder: Copy as options for steps, in-page replay, step’s context menu {: #recorder } -->
+## Recorder: 为步骤提供新的复制选项，支持页面内重放，步骤支持鼠标右键弹出菜单 {: #recorder }
 
-{% Img src="image/dPDCek3EhZgLQPGtEG3y0fTn4v82/uCqjrGj716ZbDJ4N37dl.png", alt="New copy options in the Recorder panel.", width="800", height="615" %}
+{% Img src="image/dPDCek3EhZgLQPGtEG3y0fTn4v82/uCqjrGj716ZbDJ4N37dl.png", alt="Recorder 面板支持新的复制选项", width="800", height="615" %}
 
 <!-- Open an existing user flow in the **Recorder**. Previously, when you replayed the user flow, DevTools would always start the replay by navigating to or reloading the page. -->
+打开 **Recorder** 面板里面已有的一个用户流程。在此之前，当您重放该用户流程时，DevTools 会始终通过跳转或重新加载页面的方式来执行此次的重放。
```
Comment (jecfish) [inline site/zh/blog/new-in-devtools-109/index.md:34]:
```suggestion
在 **Recorder** 面板里面打开一个用户流程。之前，当您重放该用户流程时，DevTools 会始终通过跳转或重新加载页面的方式来执行此次的重放。
```

File: site/zh/blog/new-in-devtools-109/index.md [hunk 6]
```diff
@@ -27,88 +26,118 @@ draft: true
...
 
 <!-- With the latest updates, the **Recorder** shows the navigation step separately. You can right-click and remove it to perform in-page replay!  -->
+有了这次更新，**Recorder** 面板就可以分别展示各个跳转（navigation）步骤。您可以鼠标右键并删除它来执行页面内重放！
 
 <!-- Apart from that, you can right-click a step and copy it to the clipboard in the **Recorder* panel instead of exporting the whole user flow. It works with [extensions](https://goo.gle/recorder-extension) too. For example, try to copy a step as a [Nightwatch Test](https://bit.ly/nightwatch-recorder) script. With this feature, you can update any existing script with ease. -->
+除此之外，您还可以对步骤进行鼠标右键操作并将其复制到 **Recorder** 面板的剪贴板中，而不再是导出整个用户流程。它也适用于 [扩展](https://goo.gle/recorder-extension)。例如，尝试将步骤复制为 [Nightwatch Test](https://bit.ly/nightwatch-recorder) 脚本。有了这个功能，您可以轻松地更新任何已有脚本。
```
Comment (jecfish) [inline site/zh/blog/new-in-devtools-109/index.md:40]:
```suggestion
除此之外，您还可以对步骤进行鼠标右键操作并将其复制到 **Recorder** 面板的剪贴板中，而不再是导出整个用户流程。它也适用于 [Chrome 扩展](https://goo.gle/recorder-extension)。例如，尝试将步骤复制为 [Nightwatch Test](https://bit.ly/nightwatch-recorder) 脚本。有了这个功能，您可以轻松地更新任何已有脚本。
```

File: site/zh/blog/new-in-devtools-109/index.md [hunk 7]
```diff
@@ -27,88 +26,118 @@ draft: true
...
 
 <!-- Previously, you could access the step menu only through the 3-dot button. You can now right-click anywhere on the step to access the menu. -->
+在此之前，您只能通过点击 3 个点按钮的方式来弹出步骤菜单。现在，您可以在步骤上的任何位置通过鼠标右键的方式来弹出菜单。
```
Comment (jecfish) [inline site/zh/blog/new-in-devtools-109/index.md:43]:
```suggestion
现在，您可以在步骤上的任何位置通过鼠标右键的方式来弹出菜单。在此之前，您只能通过点击 3 个点按钮的方式来弹出步骤菜单。
```

File: site/zh/blog/new-in-devtools-109/index.md [hunk 8]
```diff
@@ -27,88 +26,118 @@ draft: true
...
 
-Chromium issues: [1322313](https://crbug.com/1322313), [1351649](https://crbug.com/1351649), [1322313](https://crbug.com/1322313), [1339767](https://crbug.com/1339767)
+Chromium 议题：[1322313](https://crbug.com/1322313), [1351649](https://crbug.com/1351649), [1322313](https://crbug.com/1322313), [1339767](https://crbug.com/1339767)
 
 
 <!-- ## Show actual function names in performance’s recordings {: #performance } -->
+## Performance 面板里面的记录显示实际的函数名称 {: #performance }
```
Comment (jecfish) [inline site/zh/blog/new-in-devtools-109/index.md:49]:
```suggestion
## 在 Performance 面板里显示实际的函数名称 {: #performance }
```

File: site/zh/blog/new-in-devtools-109/index.md [hunk 9]
```diff
@@ -27,88 +26,118 @@ draft: true
...
 
 <!-- The **Performance** panel now shows the actual function names and their sources in the trace if there’s a sourcemap. -->
+如果有 sourcemap，**Performance** 面板现在将会在 trace 中显示实际的函数名称和它们的源文件。
```
Comment (jecfish) [inline site/zh/blog/new-in-devtools-109/index.md:52]:
```suggestion
**Performance** 面板现在会读取 sourcemap， 在 trace 中显示实际的函数名称和它们的源文件。
```

File: site/zh/blog/new-in-devtools-109/index.md [hunk 10]
```diff
@@ -118,45 +147,59 @@ Chromium issues: [1267690](https://crbug.com/1267690), [1246863](https://crbug.c
 
 
 <!-- ## [Experimental] Enhanced UX in managing breakpoints -->
+## 【实验】优化断点管理界面的 UX
```
Comment (jecfish) [inline site/zh/blog/new-in-devtools-109/index.md:150]:
```suggestion
## 【实验阶段】优化断点管理界面的 UX
```

File: site/zh/blog/new-in-devtools-109/index.md [hunk 11]
```diff
@@ -118,45 +147,59 @@ Chromium issues: [1267690](https://crbug.com/1267690), [1246863](https://crbug.c
...
-{% Img src="image/dPDCek3EhZgLQPGtEG3y0fTn4v82/ytfyl8qK5rkHQRTS3sXf.png", alt="Show Breakpoint pane before and after the redesign.", width="800", height="684" %}
+{% Img src="image/dPDCek3EhZgLQPGtEG3y0fTn4v82/ytfyl8qK5rkHQRTS3sXf.png", alt="对比显示重构前后的 Breakpoint 面板。", width="800", height="684" %}
 
 {# https://chromium.googlesource.com/devtools/devtools-frontend/+/f2140378e0bb1687b263c226de01b741487ff324 #}
-Chromium issues: [1346231](https://crbug.com/1346231), [1324904](https://crbug.com/1324904) 
+Chromium 议题：[1346231](https://crbug.com/1346231), [1324904](https://crbug.com/1324904)
 
 
 <!-- ## [Experimental] Automatic in-place pretty print -->
+## 【实验】自动格式化
```
Comment (jecfish) [inline site/zh/blog/new-in-devtools-109/index.md:184]:
```suggestion
## 【实验阶段】自动格式化
```

**Review by jecfish — APPROVED** [review]

**Pull request merged**